helm-chart/binderhub/templates/deployment.yaml

```diff
@@ -117,7 +117,7 @@ spec:
             name: binder
         livenessProbe:
           httpGet:
-            path: {{ default "/" .Values.config.BinderHub.base_url }}about
+            path: {{ default "/" .Values.config.BinderHub.base_url }}versions
```

Comment:
Hi @minrk! I know this PR is already merged. But I was on vacation and now have time to go over updates in BinderHub. And I am now just curious if there was a specific reason to choose `/versions` path instead of `/health`?

Reply:
If a pod fails a `liveness` check it should be killed and restarted, so the liveness check should only fail if the pod itself is unhealthy or unready. Our health endpoint describes the health of the whole application including pod quotas, etc. If we used `/health` for a liveness probe, whenever we hit a pod quota or if the Hub is restarting, the binder pod itself would be restarted. So I went with the simplest handler we have to measure only "is this pod responding to basic requests promptly". That way, it doesn't get added to rotation while it's not ready to accept http requests, but that's the only condition checked.

We could have a different health check (or modify the one we have) that truly measures only pod health and use that, but `/versions` was the closest to `/about` without serving a human-facing page that I didn't want to demote in the logs.

Reply:
Thank you!